launchdarkly-server-sdk/Cargo.toml

```diff
@@ -13,6 +13,9 @@ exclude = [
     "coverage.sh"
 ]
 
+[package.metadata.docs.rs]
+features = ["event-compression"]
```

Comment:
This should enable this feature when generating the docs for docs.rs, which will allow our docs to link to the enable compression method.

Even though this is docs only, doing it as a fix so I can push a new version of the documentation to the docs.rs site.